pandas/core/series.py

```diff
@@ -3236,7 +3246,7 @@ def nsmallest(self, n=5, keep='first'):
         Monserat        5200
         dtype: int64
 
-        The `n` largest elements where ``n=5`` by default.
+        The `n` smallest elements where ``n=5`` by default.
```

Comment:
was this a typo?

Reply:
This comes under the definition of `nsmallest`, so I hope I *corrected* a typo and didn't *make* one. If otherwise, please suggest a correction @jreback